gcloud/storage/acl.py

```diff
@@ -409,7 +419,7 @@ def _save(self, acl, predefined, client):
 
         :type predefined: string or None
         :param predefined: An identifier for a predefined ACL.  Must be one
-                           of the keys in :attr:`_PREDEFINED_ACLS`
+                           of the keys in :attr:`PREDEFINED_JSON_ACLS`
```

Comment:
`:data:` is for module-scope variables;  `PREDEFINED_JSON_ACLS` and `PREDEFINED_XML_ACLS` are class-scope attributes, and should therefore use `:attr:`.